src/main/java/com/itacademy/cms/model/UsersCourses.java

```diff
@@ -0,0 +1,38 @@
+package com.itacademy.cms.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import lombok.*;
+
+import javax.persistence.*;
+import com.itacademy.cms.model.Users;
+import com.itacademy.cms.model.Course_status;
+import com.itacademy.cms.model.Groups;
+import com.itacademy.cms.model.Courses;
+
+@Data
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users_couses")
+public class UsersCourses {
+  @EmbeddedId
+  @JsonUnwrapped
```

Comment:
is `JsonUnwrapped` really needed in persistence layer? This entity will never leave application internals (user will see DTO)